activities/tool-management/github.md

```diff
@@ -0,0 +1,21 @@
+---
+type: resource
+---
+
+# GitHub
+
+GitHub, Inc. is a subsidiary of Microsoft which provides hosting for software development and version control using Git. It offers the distributed version control and source code management (SCM) functionality of Git, plus its own features. It provides access control and several collaboration features such as bug tracking, feature requests, task management, continuous integration and wikis for every project.
+
+## Using GitHub
+
+GitHub is our bread and butter when it comes to codebase management, codebase auditing, issue tracking and version control, both for ourselves as our community.
+
+## GitHub management
+
+GitHub is managemed by the Directors and Codebase Stewards. New employees will be onboarded by one of the Codebase Stewards. We also have [documention available on About for newcomers.](../trainings/github-for-newcomers.md)
+
+## Privacy and GDPR
+
+GitHub has [an extensive Privacy Notice.](https://docs.github.com/en/free-pro-team@latest/github/site-policy/github-privacy-statement)
```

Comment:
The point of the privacy and GDPR section was to explain:

- what data the tool collects and processes
- where
- if it's processed outside the EU, what the contractual mechanism in place is to ensure Europeans' personal data is appropriately protected (so that **we** can be confident our use of that tool is GDPR compliant)

The reason for copying the section of the tool's privacy policy was so that we could clearly see when the content had changed (so we'd have a record and we could be confident we were GDPR compliant - we're liable if we contract with a non-compliant tool and ship our users' personal data abroad).

Older tool pages (written when this info was housed under a separate privacy section) show this pattern clearly:

- [Google Workspace](https://about.publiccode.net/activities/tool-management/google-workspace.html)
- [Docusign](https://about.publiccode.net/activities/tool-management/docusign.html)